DOC-4766 Update webinar link (#421)
couchbase/docs-server: modules/manage/pages/manage-security/manage-connections-and-disks.adoc

 ** Windows: `C:\Program Files\couchbase\server\var\lib\couchbase\isasl.pw` and `C:\Program Files\couchbase\server\var\lib\couchbase\var\lib\config\`.

> For more information, see the webinar provided at

Reviewer: ```suggestion
You can learn more about encryption in the webinar 
```
This is just some alternative language that I thought worked a little better. The current statement feels more like you're going to get some reference information instead of tutorial-style info. I also tend to avoid saying "at" followed by link text. If it were just the raw hyperlink, then I think "at" works better.

> {url-webinar}[Understanding Database Encryption with Couchbase and Vormetric].

Reviewer: ```suggestion
{url-webinar}[Understanding Database Encryption with Couchbase & Vormetric^].
```
Updated link text to match exact webinar title; added caret so link opens in new tab
